features/step_definitions/sign_in_steps.rb

```diff
@@ -0,0 +1,13 @@
+Given(/^I have an account$/) do
```

Comment:
`I have an account` means the user is registered in the system. Why does the user have to sign out? 
A more descriptive title for this would be 'Given I sign up for an account'